lib/hal/led.h

```diff
@@ -0,0 +1,18 @@
+#ifndef HAL_LED_H
+#define HAL_LED_H
+
+#include <hal/xbox.h>
+#include <xboxkrnl/xboxkrnl.h>
```

Comment:
These don't seem to be used here

Reply:
`<hal/xbox.h>` is probably not used, `<xboxkrnl/xboxkrnl.h>` is used for `HalWriteSMBusValue()`.

Reply:
But not in the header. It should only be present in the source file.

*Otherwise you increase compilation time and clutter the namespace with typenames and functions which possibly conflict with others.*

Reply:
Oh, TIL. :+1: